tests/test_train.py

```diff
@@ -0,0 +1,43 @@
+import os
+
+import pytest
+
+from ecodse_funtime_alpha.train import get_args
+
+
+class TestArgparse(object):
+
+    def test_argparsenormal(self):
+        fakearg = ['--imagepath=./', '--labelpath=fakedir/name.csv',
+                   '--seed=1', '--kernels=10', '--ksize=1',
+                   '--lr=0.01', '--nepoch=2', '--batchsize=4'
+                   ]
+        args = get_args(fakearg)
+        assert args.imagepath == './'
+        assert args.labelpath == 'fakedir/name.csv'
+        assert args.seed == 1
+        assert args.kernels == 10
+        assert args.ksize == 1
+        assert args.lr == 0.01
+        assert args.nepoch == 2
+        assert args.batchsize == 4
+
+    @pytest.mark.xfail(raises=SystemExit)
+    def test_argparse_lr(self):
+        fakearg = ['--lr=a']
+        _ = get_args(fakearg)
```

Comment:
You should assert that the proper exception is thrown.

https://stackoverflow.com/questions/23337471/how-to-properly-assert-that-an-exception-gets-raised-in-pytest